<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QWidget" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="leftMargin">
                <number>1</number>
               </property>
               <property name="topMargin">
                <number>1</number>
               </property>
               <property name="rightMargin">
                <number>1</number>
               </property>
               <property name="bottomMargin">
                <number>1</number>
               </property>
               <item>
                <widget class="QTextBrowser" name="msgBrowser"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="leftMargin">
                <number>1</number>
               </property>
               <property name="topMargin">
                <number>1</number>
               </property>
               <property name="rightMargin">
                <number>1</number>
               </property>
               <property name="bottomMargin">
                <number>1</number>
               </property>
               <item>
                <widget class="QFontComboBox" name="fontCbx"/>
               </item>
               <item>
                <widget class="QComboBox" name="sizeCbx">
                 <property name="currentIndex">
                  <number>4</number>
                 </property>
                 <item>
                  <property name="text">
                   <string>8</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>9</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>10</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>11</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>12</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>13</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>14</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>15</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>16</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>17</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>18</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>19</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="boldTBtn">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>加粗</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-image: url(:/images/jiachu.png);</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="italicTBtn">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>加粗</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-image: url(:/images/xieti.png);</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="underlineTBtn">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>加粗</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-image: url(:/images/xiahuaxian.png);</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="colorTBtn">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>加粗</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-image: url(:/images/yanse.png);</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="saveTBtn">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>加粗</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-image: url(:/images/baocun.png);</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="clearTBtn">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>加粗</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-image: url(:/images/qingchu.png);</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="leftMargin">
                <number>1</number>
               </property>
               <property name="topMargin">
                <number>1</number>
               </property>
               <property name="rightMargin">
                <number>1</number>
               </property>
               <property name="bottomMargin">
                <number>1</number>
               </property>
               <item>
                <widget class="QTextEdit" name="msgTextEdit"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="usrTblWidget">
           <column>
            <property name="text">
             <string>用户名</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="sendBtn">
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="usrNumLbl">
           <property name="text">
            <string>在线人数：0人</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="exitBtn">
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
